spectacles/cli.py

```diff
@@ -273,6 +294,13 @@ def _build_base_subparser() -> argparse.ArgumentParser:
         help="Display debug logging during spectacles execution. \
             Useful for debugging and making bug reports.",
     )
+    base_subparser.add_argument(
+        "--log-directory",
```

Comment:
`--log-dir` work for this? Think it's a common enough abbreviation I'd be okay with it for convenience.